tests/aws/services/lambda_/event_source_mapping/test_lambda_integration_kinesis.validation.json

```diff
@@ -75,3 +73,9 @@
     "last_validated_date": "2024-10-12T13:23:12+00:00"
   }
 }
+"
```

Comment:
Why is there a `"` in here?

Reply:
Good catch, the entire validations file was messed up 😆 
Re-generated from scratch ✅